metricflow/cli/utils.py

```diff
@@ -234,3 +234,25 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:  # type: ignore[misc]
             exit(1)
 
     return wrapper
+
+
+def dbt_project_file_exists() -> bool:
+    """Check that the cwd is a dbt project root. Currently done by checking for existence of dbt_project.yml."""
+    return pathlib.Path("dbt_project.yml").exists()
+
+
+def error_if_not_in_dbt_project(func: Callable) -> Callable:
+    """Decorator to output an error message and exit if caller is not in a root directory of a dbt project."""
```

Comment:
Nifty, this way we can update the check as more of this functionality is exposed from dbt-core.